pkgs/samples/lib/default_dart.dart

```diff
@@ -1,5 +1,9 @@
+// Copyright (c) 2024, the Dart project authors.  Please see the AUTHORS file
```

Comment:
One drawback of having the copyright header here (and for default_flutter.dart) is that they'll show up for most dartpad snippets. I think it's ok to omit here, but to keep it for the more specific samples (sunflower, ...).

Reply:
In this PR, I ended up stripping the copyright from each snippet when loaded into DartPad, but keeping it in the source. Is there any concern with that? As the license comments end up distracting from the samples anyway.

Reply:
Ah, I'd missed that! No real concerns.